simple_ddl_parser/parser.py

```diff
@@ -348,7 +346,7 @@ def run(
             Dict == one entity from ddl - one table or sequence or type.
         """
         if output_mode not in dialect_by_name:
-            raise SimpleDDLParserException(
```

Comment:
I am not sure, if someone uses this exception in their code (to catch it, for instance). If, it is possible, then it is more correct to make an alias and deprecate it officially.

Reply:
Do not rename exceptions, I didn’t merge it because of those changes. Sorry for long delay with review. Didn’t have any tome

Reply:
Fixed

Reply:
I think, there is no reason to exist both `SimpleDDLParserException` and `DDLParserError` exception type. Only backward compatibility.